src/content/index.tsx

```diff
@@ -1,18 +1,25 @@
+import FinalPage from "../components/FinalPage";
+import StartExperiment from "../components/StartExperiment";
 import { A, AB, B } from "../components/agent-symbols";
 import { ContentPanelProps } from "../components/main-layout/ContentPanel";
 import Definition from "../components/shared/Definition";
-import { Module } from "../types";
+import { MICRO } from "../constants";
+import { LayoutType, Module, PageContent, Section } from "../types";
 
-export const highAffinityContentArray: ContentPanelProps[] = [
+export const highAffinityContentArray: PageContent[] = [
     // making the content array 1 indexed to match the page numbers
     {
         content: "",
+        section: Section.Introduction,
+        layout: LayoutType.FullScreen,
```

Comment:
This granularity in defining the `PageContent` is reminiscent of gatsby and I really like it. 

Down the road it could evolve into a great API for people building education modules who don't want to get into the weeds on the code.